engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/inputSettings/ChangeBindingPopup.java

```diff
@@ -15,9 +15,11 @@
  */
 package org.terasology.rendering.nui.layers.mainMenu.inputSettings;
 
+import org.lwjgl.Sys;
 import org.terasology.assets.ResourceUrn;
 import org.terasology.config.BindsConfig;
 import org.terasology.config.Config;
+import org.terasology.config.InputConfig;
```

Comment:
This import doesn't seem to be required for anything. [4]

(Use Ctrl+Alt+O to remove unused imports in IDEA!)